system-tests/services/_scripts/setup-external-volumes

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# create simple app
+cat <<EOF | dcos marathon app add
```

Comment:
unfortunately that didnt help. script is executed and no error is reported, but it doesnt work.